docs/API.md

```diff
@@ -10,33 +10,51 @@ The REST API has two endpoints: /relay and /balance.
 
 Relays the supplied transaction, returns a signed receipt that [guarantees](./guarantees.md) transaction submission.
 
-| Relay | |
-| --- | --- |
-| Endpoint | /relay |
-| Method | POST |
-| Content-Type | application/json |
-| Request body format | [Relay transaction](./relayTransaction.md) |
-| Response body format | [Relay receipt](./relayReceipt.md) |
+| Relay                |                                            |
+| -------------------- | ------------------------------------------ |
+| Endpoint             | /relay                                     |
+| Method               | POST                                       |
+| Content-Type         | application/json                           |
+| Request body format  | [Relay transaction](./relayTransaction.md) |
+| Response body format | [Relay receipt](./relayReceipt.md)         |
 
 #### Ordering
 
 any.sender guarantees that transactions from the same address will be mined in the order in which it receives them. Therefore, if you need to guarantee a specific order for transactions, then you should send and receive them serially. Only send a second transaction once you've received an acknowledgment for the first from the any.sender API.
 
+#### Error codes
+
+In most cases, you should expect the 200 OK response from any.sender that the job was accepted. However, there is a range of potential error codes that can be returned if the job is malformed or if there is a server-side issue.
```

Comment:
I think we should change the language here - we dont need to say that it'll be accepted in most cases. Maybe just write:
```
The api can return the following HTTP status codes:
```

Reply:
done